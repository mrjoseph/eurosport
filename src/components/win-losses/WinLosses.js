import React from 'react';
import calculateScore from '../../utils/calculateScore';
import './winlosses.css';

const WinsLosses = ({ score }) => {
  const arr = calculateScore(score);
  return (
      <div className="win-loss">
        <div>Wins: {arr[0]}</div>
        <div>Losses: {arr[1]}</div>
      </div>
  );
};

export default WinsLosses;
