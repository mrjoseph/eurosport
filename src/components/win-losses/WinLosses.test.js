import React from 'react';
import renderer from 'react-test-renderer';
import WinLosses from './WinLosses';
describe('Wins and Losses', () => {
  it('renders wins and losses component ', () => {
    const score = [1,0,0,0,1];
    const rendered = renderer.create(
        <WinLosses score={score} />
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
