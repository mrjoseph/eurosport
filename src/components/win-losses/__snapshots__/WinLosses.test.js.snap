// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wins and Losses renders wins and losses component  1`] = `
<div
  className="win-loss"
>
  <div>
    Wins: 
    2
  </div>
  <div>
    Losses: 
    3
  </div>
</div>
`;
